gnupg-hook

```diff
@@ -15,15 +15,15 @@ run_hook() {
     }
 
     get_file() {
-        IFS=: read ckdev ckarg1 ckarg2 ckarg3 <<EOF
+        IFS=: read -r ckdev ckarg1 ckarg2 ckarg3 <<EOF
```

Comment:
It is probably better to keep the `read` default behavior here, since the arguments are filled on the kernel command line, I want to allow the "user" to input escaped characters.